sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs

```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
+        public static bool operator !=(Azure.ResourceManager.DevCenter.Models.DevCenterHibernateSupport left, Azure.ResourceManager.DevCenter.Models.DevCenterHibernateSupport right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class DevCenterImageReference
+    {
+        public DevCenterImageReference() { }
+        public string ExactVersion { get { throw null; } }
+        public Azure.Core.ResourceIdentifier Id { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct DevCenterLicenseType : System.IEquatable<Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public DevCenterLicenseType(string value) { throw null; }
+        public static Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType WindowsClient { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType left, Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType left, Azure.ResourceManager.DevCenter.Models.DevCenterLicenseType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class DevCenterNetworkConnectionPatch : Azure.ResourceManager.DevCenter.Models.DevCenterTrackedResourceUpdate
+    {
+        public DevCenterNetworkConnectionPatch() { }
+        public string DomainName { get { throw null; } set { } }
+        public string DomainPassword { get { throw null; } set { } }
+        public string DomainUsername { get { throw null; } set { } }
+        public string OrganizationUnit { get { throw null; } set { } }
+        public Azure.Core.ResourceIdentifier SubnetId { get { throw null; } set { } }
+    }
+    public partial class DevCenterOperationStatus : Azure.ResourceManager.Models.OperationStatusResult
+    {
+        internal DevCenterOperationStatus() : base (default(string)) { }
+        public System.BinaryData Properties { get { throw null; } }
+        public string ResourceId { get { throw null; } }
```

Comment:
Is this a `arm-id`?

Reply:
Yes, the config in autorest.md was commented out accidentally, fixed now.